### PR TITLE
Externalize logging pattern

### PR DESCRIPTION
**Pull request by paraita**

### CODE REVIEW

**Comment by paraita** [comment]
retest this please

**Comment by paraita** [comment]
retest this please

**Comment by paraita** [comment]
retest this please

File: scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/task/Log4JTaskLogs.java [hunk 1]
```diff
@@ -79,8 +80,16 @@
 
     /** Default layout for logs */
     public static Layout getTaskLogLayout() {
-        return new PatternLayout("[%X{" + Log4JTaskLogs.MDC_JOB_ID + "}t%X{" + Log4JTaskLogs.MDC_TASK_ID + "}@%X{" + Log4JTaskLogs.MDC_HOST +
-            "};%d{HH:mm:ss}]" + " %m %n");
+        PatternLayout patternLayout;
+        if (PASchedulerProperties.SCHEDULER_JOB_LOGS_PATTERN.isSet()) {
+            patternLayout = new PatternLayout(PASchedulerProperties.SCHEDULER_JOB_LOGS_PATTERN.getValueAsString());
+        }
+        else {
```
Comment (lpellegr) [inline scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/task/Log4JTaskLogs.java]: Formatting: _else_ on the same line as the closing curly brace of the _if_.

**Pull request merged**